src/components/NavigationGroup/NavigationGroup.js

```diff
@@ -15,7 +15,12 @@ const NavigationGroup = ({ crumbs, additionalNav }) => {
 };
 
 NavigationGroup.propTypes = {
-  crumbs: PropTypes.arrayOf(PropTypes.string),
+  crumbs: PropTypes.arrayOf(
```

Comment:
`PropTypes.array` would suffice in this file, the component is only passing the array on, and does nothing with the items of the array.